code/game/machinery/computer/arcade/arcade.dm

```diff
@@ -0,0 +1,75 @@
+/obj/machinery/computer/arcade
+	name = "arcade machine"
+	desc = "Does not support pinball."
+	icon = 'icons/obj/computer.dmi'
+	icon_state = "arcade"
+	circuit = "/obj/item/weapon/circuitboard/arcade"
+	var/datum/arcade_game/game
+	machine_flags = EMAGGABLE | SCREWTOGGLE | CROWDESTROY | WRENCHMOVE | FIXED2WORK
+	emag_cost = 0 // because fun
+	light_color = LIGHT_COLOR_GREEN
+
+
+/obj/machinery/computer/arcade/New()
+	..()
+	game = new /datum/arcade_game/space_villain(src)
```

Comment:
You need to qdel/null this in `Destroy()`

Reply:
done